src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
```

Comment:
The `alt` attribute for the img tag should be more descriptive. It's better to describe what's on the picture rather than just writing 'Moyo logo'. For example, it can be 'Moyo company logo'.